kamel-core/src/commonMain/kotlin/io/kamel/core/config/KamelConfigBuilder.kt

```diff
@@ -48,7 +48,8 @@ public class KamelConfigBuilder {
     }
 
     public fun <I : Any, O : Any> mapper(mapper: Mapper<I, O>) {
-        mappers[mapper.inputKClass] = mapper as Mapper<Any, Any>
+        mappers[mapper.inputKClass] =
+            (mappers[mapper.inputKClass] ?: emptyList()) + (mapper as Mapper<Any, Any>)

```

Comment:
actually you can use `getOrPut` here too

Reply:
Had to make the list of mappers a `MutableList`, though.